src/components/pages/PendingApplications/PendingApplication.js

```diff
@@ -4,15 +4,30 @@ import ApplicationModal from './ApplicationModal';
 import { Table, Button, Tag } from 'antd';
 import './PendingApplication.css';
 
+// Filter by status
+const statusFilter = (value, record) => {
+  if (Array.isArray(value)) {
+    return (
```

Comment:
Might be a good case for array 'some' method:
`return value.some(val => val === true)`